external_search.py

```diff
@@ -763,7 +809,7 @@ def v4_body(cls):
         licensepool_fields_by_type = {
             'integer': ['collection_id', 'data_source_id'],
             'date': ['availability_time'],
-            'boolean': ['availability', 'open_access', 'suppressed', 'licensed'],
+            'boolean': ['availabile', 'open_access', 'suppressed', 'licensed'],
```

Comment:
This is the change you mentioned earlier today; I put it in this branch by mistake.

Reply:
Now you spelled it wrong :)